src/openrct2/config/Config.cpp

```diff
@@ -511,8 +511,8 @@ namespace Config
         if (reader->ReadSection("font"))
         {
             auto model = &gConfigFonts;
-            model->FileName = reader->GetCString("file_name", nullptr);
-            model->FontName = reader->GetCString("font_name", nullptr);
+            model->FileName = reader->GetString("file_name", nullptr);
```

Comment:
Put nullptr to ""

Reply:
Damn, I was so focused on the ones in IniReaderTest that I completely overlooked these :/